view-admin/src/styles/Home.module.css

```
@@ -174,6 +174,16 @@
   color: #fff;
 }
 
+.not_hober_Button{
```

```suggestion
.not_hover_Button{
```
私のコメントがミスってた。hoverです。。。